packages/s2-core/src/common/constant/frozen.ts

```diff
@@ -1,28 +1,27 @@
-export enum FrozenCellType {
-  ROW = 'row',
-  COL = 'col',
-  TRAILING_ROW = 'trailingRow',
-  TRAILING_COL = 'trailingCol',
-  SCROLL = 'scroll',
-  TOP = 'top',
-  BOTTOM = 'bottom',
+export enum FrozenGroupPosition {
```

Comment:
area 是不是要好点, position 感觉有点歧义

```suggestion
export enum FrozenGroupArea {
```